<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookMage</class>
 <widget class="QMainWindow" name="BookMage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/concreto.jpg);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>19</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="BolaFogo">
       <property name="text">
        <string>Bola de Fogo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="Desintegracao">
       <property name="text">
        <string>Desintegração</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="Invisibilidade">
       <property name="text">
        <string>Invisibilidade</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="RaioEletrico">
       <property name="text">
        <string>Raio Elétrico</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="Telecinese">
       <property name="text">
        <string>Telecinese</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radioButton_9">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="radioButton_10">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioButton_11">
       <property name="text">
        <string>Quebra de encantos</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
